testsuites/CBLTester/CBL_Functional_tests/conftest.py

```diff
@@ -490,6 +503,11 @@ def params_from_base_test_setup(request, params_from_base_suite_setup):
     db = None
     cbl_db = None
     if create_db_per_test:
+        if enable_file_logging and liteserv_version >= "2.5.0":
+            cbllog = FileLogging(base_url)
+            cbllog.configure(log_level="verbose", max_rotate_count=2,
```

Comment:
can we have something like enable_file_logging=verbose or debug or something else? Whatever is set, it sets log_level? I know you need another verification whether valid log_levels are added or not. We don't really care about other log levels. If it is too hard, you can leave as it is

Reply:
basically verbose is enabled only for debug builds. For release build verbose and debug are same. Hence I provided verbose as log level